README.md

```diff
@@ -46,42 +50,113 @@ If you're using Docker / Kubernetes+Docker you'll need to override the ipv4 ip f
       net.ipv4.ip_forward: 1
 ```
 
+### sysctl - vm.mmap_rnd_bits
+
+We are setting this sysctl to a default of `32`, some systems only support smaller values and this will generate an error. Unfortunately we cannot determine the correct aplicable maximum. If you encounter this error you have to override this sysctl in your playbook.
```

Comment:
```suggestion
We are setting this sysctl to a default of `32`, some systems only support smaller values and this will generate an error. Unfortunately we cannot determine the correct applicable maximum. If you encounter this error you have to override this sysctl in your playbook.
```